spec/features/employee/administrator/user/administrator_edit_user_spec.rb

```diff
@@ -0,0 +1,39 @@
+feature = <<~HEREDOC
+  When administrator wants to edit user valid credentials,
+  he wants to edit just password of user by new and confirmationed password,
+  he wants to edit others credentials of user without entering a password,
+  so that user have beening  new credentials
+HEREDOC
+
+RSpec.feature feature, issues: ['railgun#151'] do
+  before { login_as FactoryGirl.create(:administrator) }
+  let(:user) { create(:user) }
+  before { visit edit_staff_user_path(user) }
+
+  scenario = <<~HEREDOC
+    Given administrator is on the edit user page
+    When he fills in email 'new@mail.com' with no entering a password and clicks 'Update User' button
```

Comment:
When he fills in email 'new@mail.com' and clicks 'Update User' button - this is enough.